src/app/convai-checker.component.spec.ts

```diff
@@ -205,13 +205,74 @@ class ConvaiCheckerWithAttributeInputTestComponent {
   }
 }
 
+@Component({
+  selector: 'my-comp-hide-loading-icon-after-load-setting',
```

Comment:
The "my-comp" in the name seems a bit odd...is that normal or should it be "convai-checker-comp"

Reply:
I mean, I did that for all the others in the test...it's not convai-checker because it's a wrapper component for testing. I guess I could rename everything, but these are already so verbose. If I had to rename I'd probably just go with test-comp instead of my-comp which I think adds limited value. Thoughts?

Reply:
my-comp gives the feeling of a copy+paste that didn't get checked.  I think it would look a lot better as "test-comp" or even just "test" going forward.

Reply:
Done.